google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -178,19 +175,18 @@ def __init__(
         )
 
         pipeline_name = pipeline_job["pipelineSpec"]["pipelineInfo"]["name"]
-        job_id = job_id or "{pipeline_name}-{timestamp}".format(
+        self.job_id = job_id or "{pipeline_name}-{timestamp}".format(
```

Comment:
I know this is existing code, but why isn't there a way to get the `job_id` directly from the service instead of deriving it in the client SDK. 

This seems like it could break if the service changes how job ID's are generated.

Reply:
I think `job_id` was provided by the user, or generated by the system and only used during `run` command but not the instance generation process.  That's why we need to store it somewhere, since it wasn't used during the instance creation. 

`Name` shouldn't be passed in since it was an output-only field and was wrongly passed in previously. Although the services generated the name using the same way.